### DISCUSSION TITLE
Some tab / button trigger by js not working

### DISCUSSION THREAD

**Discussion by erictioh** [gildas-lormeau/SingleFile#1483] [Q&A]
**Describe the bug**
Go to this site
https://squidfunk.github.io/mkdocs-material/reference/math/

And then save as singlefile html. Notice that the following 3 portion does not work. please check the screenshot.

<img width="1374" alt="image" src="https://github.com/gildas-lormeau/SingleFile/assets/44055730/4eb33db1-4299-4f2b-9f20-fabb2a499101">

As i trace the 3 components, they are triggered by event into js function, however i'm sure that in the singlefile setting javascript is included.
<img width="1381" alt="image" src="https://github.com/gildas-lormeau/SingleFile/assets/44055730/fe84725c-4cc4-4add-aae2-2a500a2f4078">
<img width="1712" alt="image" src="https://github.com/gildas-lormeau/SingleFile/assets/44055730/5186a8c3-d452-4f00-b341-22d645638815">

Here is the setting where the js is not removed. meaning included. but somehow this kind of js is not being included despite this setting is on. Not sure why

<img width="631" alt="image" src="https://github.com/gildas-lormeau/SingleFile/assets/44055730/31d3e1e9-67e5-4189-8494-8af0b6443b81">

**Comment by gildas-lormeau** [comment]
I can confirm the existence of this problem. It's part of SingleFile's limitations. This web page was not designed to be saved and consulted as an archive. That's why it doesn't work unfortunately.

**Comment by erictioh** [comment]
but i tried in firefox it works, why? it foesnt work with chrome

**Comment by gildas-lormeau** [comment]
Actually I was wrong, you can save the page in Firefox and Chrome. For this you have to disable the following options:
 - `HTML content > compress HTML content`
 - `HTML content > remove hidden elements`
 - `Stylesheets > remove unused styles`
 - `Stylesheets > remove stylesheets for alternative devices to screens`
 - `Images > group duplicate images together`
 - `Images > remove images for alternative screen resolutions`
 - `Fonts > remove unused fonts`
 - `Fonts > remove alternative fonts`
 - `Network > blocked resources > scripts`

You have also to enable `HTML content > save raw page`.

Here is in the ZIP file a settings file with these changes applied: [singlefile-settings.zip](https://github.com/user-attachments/files/15872034/singlefile-settings.zip)
